File: buerguer-pig/HamburgesasEIngredientes-Java-IvanRC/src/main/java/dto/ingrediente/IngredienteDto.java
package dto.ingrediente;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Root;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Root(name = "ingrediente")
public class IngredienteDto {
    @Attribute(name = "id")
    private String id;

    @Attribute(name = "nombre")
    private String nombre;

    @Attribute(name = "precio")
    private String precio;
}

File: buerguer-pig/SergioDeIscar-restaurante-kotlin/RestauranteFicheros/src/main/kotlin/dto/IngredienteDto.kt
package dto

import org.simpleframework.xml.Attribute
import org.simpleframework.xml.Root

@Root(name = "ingrediente")
data class IngredienteDto(
    @field:Attribute(name = "id")
    @param:Attribute(name = "id")
    val id: String,
    @field:Attribute(name = "nombre")
    @param:Attribute(name = "nombre")
    val nombre: String,
    @field:Attribute(name = "precio")
    @param:Attribute(name = "precio")
    val precio: String
)